### PR TITLE
Mobile menu toggle

### PR DESCRIPTION
**Pull request by ciar4n**
Basic styling

### CODE REVIEW

File: src/css/components/navbar.css [hunk 1]
```diff
@@ -4,17 +4,31 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  flex-wrap: wrap;
+
+  @media (min-width: 759px) {
```
Comment (C-Lodder) [inline src/css/components/navbar.css]: Can you can use `@media (--hiq-upper-font-range) { ... }` which is HiQ's media query?

File: src/css/components/navbar.css [hunk 2]
```diff
@@ -27,4 +41,78 @@
     margin-bottom: -2px;
     border-bottom: 2px solid var(--hiq-color-primary);
   }
+
+  @media (min-width: 759px) {
```
Comment (C-Lodder) [inline src/css/components/navbar.css]: same here

File: src/css/components/navbar.css [hunk 3]
```diff
@@ -27,4 +41,78 @@
...
+      top: -12px;
+    }
+
+    &::after {
+      top: 12px;
+    }
+  }
+
+  span,
+  span:after,
```
Comment (C-Lodder) [inline src/css/components/navbar.css]: `::`

File: index.php [hunk 4]
```diff
@@ -36,6 +37,18 @@
 	HTMLHelper::_('stylesheet', 'switch.css', ['version' => 'auto', 'relative' => true]);
 }
 
+// Mobile menu toggle
+$doc->addScriptDeclaration("
+	function menuToggle() {
```
Comment (C-Lodder) [inline index.php]: There is an inline script block at the bottom of the file already? Could you put it down there please?

File: index.php [hunk 5]
```diff
@@ -16,6 +16,7 @@
 /** @var JDocumentHtml $this */
 
 $app  = Factory::getApplication();
+$doc = Factory::getDocument();
```
Comment (C-Lodder) [inline index.php]: Can be removed once comment below is resolved

**Comment by C-Lodder** [comment]
@ciar4n Thanks for doing this. Just left a couple of comments

**Comment by ciar4n** [comment]
@C-Lodder Should be ok now.

**Comment by C-Lodder** [comment]
Thanks Ciaran, much appreciated

**Pull request merged**